### DISCUSSION TITLE
How to change the default search and output language from Luxembourgish to English

### DISCUSSION THREAD

**Discussion by weasam** [omkarcloud/google-maps-scraper#65] [Q&A]
I notice that default search is Luxembourgish of the output CSV file, I use command line main.py,
Anyone know how to change it from Luxembourgish to English?

**Comment by Chetan11-dev** [comment]
See https://github.com/omkarcloud/google-maps-scraper/blob/master/advanced.md#-how-to-change-the-language-of-output
